src/runtime/runtime_data.cs

```diff
@@ -120,8 +120,8 @@ private static void RestoreRuntimeDataImpl()
 
             var objs = RestoreRuntimeDataObjects(storage.GetStorage("objs"));
             RestoreRuntimeDataModules(storage.GetStorage("modules"));
-            var clsObjs = ClassManager.RestoreRuntimeData(storage.GetStorage("classes"));
             TypeManager.RestoreRuntimeData(storage.GetStorage("types"));
+            var clsObjs = ClassManager.RestoreRuntimeData(storage.GetStorage("classes"));
```

Comment:
Does the order matter or can this change be reverted?

Reply:
The order matters. Because we're re-initializing the `ClassObject`s we need the types to be present or they'll be re-created then clobbered